packages/simple-hub/.env.production

```diff
@@ -0,0 +1,12 @@
+FIREBASE_PROJECT=rock-paper-scissors-production
```

Comment:
Do we need to begin this new package with Firebase baked in?

Reply:
Would you mind clarifying? Do you mean defining the firebase project as a constant instead of an environment variable?

Reply:
Or do you mean abstracting away firebase into a seperate layer and/or package?